rllib/core/rl_module/rl_module.py

```diff
@@ -164,8 +188,16 @@ def forward_exploration(
     def _forward_exploration(self, batch: NestedDict, **kwargs) -> Mapping[str, Any]:
         """Forward-pass during exploration. See forward_exploration for details."""
 
-    @check_specs(input_spec="input_specs_train", output_spec="output_specs_train")
-    def forward_train(self, batch: SampleBatchType, **kwargs) -> Mapping[str, Any]:
+    # @check_specs(input_spec="input_specs_train", output_spec="output_specs_train")
+    def forward_train(
+        self,
+        batch: SampleBatchType,
+        filter: bool = True,
```

Comment:
Can you add descriptions for these new args to the docstring?
I have no idea what they mean. :)

Reply:
it was for debugging, they are removed.